src/Serializer/EnvelopItems/MetricsItem.php

```diff
@@ -110,4 +115,27 @@ public static function toEnvelopeItem(Event $event): string
             $statsdPayload
         );
     }
+
+    private static function replaceTagValueCharacters(string $tagValue): string
```

Comment:
Not sure if relevant for PHP, but only relay control characters are replaced as well: https://github.com/getsentry/relay/blob/3208e3ce5b1fe4d147aa44e0e966807c256993de/relay-metrics/src/protocol.rs#L62C7-L62C20

Reply:
Does not apply to any SDKs, we do this in Relay only.